regulations/templates/regulations/layers/footnote.html

```diff
@@ -1,5 +1,5 @@
 {% comment %}
-  As a first pass, footnotes are displayed as hover text
+  Footnotes are displayed as hover text and as links to the footnote text.
   Many of the styles here match regulations/layers/sxs-footnotes.html
 {% endcomment %}
-<sup><span title="{{ note }}">[{{ ref }}]</span></sup>
+<sup><a id="footnote-jump-{{ref}}" class="footnote-jump-link standard" href="#footnote-{{ref}}"><span title="{{ note }}">[{{ ref }}]</span></a></sup>
```

Comment:
I think we can remove the `span` altogether now, moving it's `title` onto the `a`. Do you agree?